bundles/binding/org.openhab.binding.tcp/src/main/java/org/openhab/binding/tcp/protocol/internal/TCPBinding.java

```diff
@@ -267,14 +267,18 @@ protected String transformResponse(String transformation, String response) {
             String transformationType = parts[0];
             String transformationFunction = parts[1];
 
-            TransformationService transformationService = TransformationHelper
-                    .getTransformationService(TCPActivator.getContext(), transformationType);
-            if (transformationService != null) {
-                transformedResponse = transformationService.transform(transformationFunction, response);
+           if (transformationType == "") {     // test for empty type first to avoid the WARN from getTransformationService
```

Comment:
You should always compare strings using the `equals` method in Java.  Testing using the `==` method might only return `true` accidentally.